grammar/FuncTestCaseLexer.g4

```diff
@@ -0,0 +1,103 @@
+lexer grammar FuncTestCaseLexer;
+
+import SubstraitLexer;
+
+options {
+    caseInsensitive = true;
+}
+
+Whitespace    : [ \t\n\r]+ -> channel(HIDDEN) ;
+
+SubstraitScalarTest: '### SUBSTRAIT_SCALAR_TEST:';
+SubstraitInclude: '### SUBSTRAIT_INCLUDE:';
+
+FormatVersion
+    : 'v' DIGIT+ ('.' DIGIT+)?
+    ;
+
+DescriptionLine
+    : '# ' ~[\r\n]* '\r'? '\n'
+    ;
+
+ErrorResult: '<!ERROR>';
+UndefineResult: '<!UNDEFINED>';
+Overflow: 'overlfow';
```

Comment:
overflow, probably should use consistent case as other literals nearby (all upper or lower)

Reply:
fixed it.